packages/core/admin/ee/admin/src/pages/SettingsPage/pages/ReviewWorkflows/selectors.ts

```diff
@@ -1,10 +1,16 @@
 import { createSelector } from '@reduxjs/toolkit';
 import isEqual from 'lodash/isEqual';
 
+import { Stage } from '../../../../../../../shared/contracts/review-workflows';
+
 import { REDUX_NAMESPACE } from './constants';
-import { initialState } from './reducer';
+import { State, initialState } from './reducer';
+
+type Store = {
+  [REDUX_NAMESPACE]: State;
+};
```

Comment:
Do you utilise the Admin store at all? We have a `RootState` interface you can extend so you have the complete store.